prestissimo/coordinator/node.properties

```diff
@@ -0,0 +1,2 @@
+node.environment=2509758e5a6160ddf8c3014651b64006eec5213a
+node.data-dir=/var/lib/presto/data
```

Comment:
On Linux writing to this dir requires root permissions. Perhaps it should be something like
`/tmp/presto/data/coordinator`

Reply:
Done